src/test/java/uk/ac/ebi/atlas/experimentpage/metadata/CellMetadataDaoIT.java

```diff
@@ -1,6 +1,5 @@
 package uk.ac.ebi.atlas.experimentpage.metadata;
 
-import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableSet;
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.BeforeAll;
```

Comment:
This class has two methods `validCellIdHasMetadataValues` and `validCellIdsHaveMetadataValues`. Isn't these two methods overlapping and should be merged in one?

Reply:
They are testing different methods, but the names carry semantics of the old method, when we were passing multiple cell IDs. I’ve changed the name accordingly.